operations/network/network_management_install_guide/aruba/mclag_power_failure.md

```diff
@@ -0,0 +1,16 @@
+# VSX: Member Power Failure
+
+
+The intent here is to show case a complete member failure scenario where spine-01 switch is completely down. 
+
+In below a visualization of the powered down spine-01:
+
+![](../docs-csm/operations/network/network_management_install_guide/img/member_power_failure.png)
```

Comment:
Is this path working? not able to see the picture.

Reply:
I dont think the path is working yet since it is not pushed to repo. I'll see if I can fix it.